docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -5,7 +5,7 @@
 <titleabbrev>Troubleshooting</titleabbrev>
 ++++
 
-The Alerting framework provides many options for diagnosing problems with Rules and Connectors.
+The {kib} Alerting framework provides many options for diagnosing problems with Rules and Connectors.
```

Comment:
Is there a difference between Alerting and "the Alerting framework"?

Reply:
They are the same things.

Reply:
We only bold terms that appear on the UI, but if the terms do not appear on the UI, it's best to not apply any style in the docs to make it easier on the human eye to read.